src/Resources/contao/classes/DataContainer.php

```diff
@@ -813,6 +877,143 @@ protected function generateGlobalButtons()
 		return $return;
 	}
 
+
+	/**
+	 * Initialize the picker
+	 *
+	 * @throws InternalServerErrorException
+	 */
+	protected function initPicker()
+	{
+		if (!isset($_GET['target']) || \Input::get('act') == 'select' || \Input::get('act') == 'paste')
+		{
+			return;
+		}
+
+		$menuBuilder = \System::getContainer()->get('contao.menu.picker_menu_builder');
+
+		if (!$menuBuilder->supports($this->strTable))
+		{
+			return;
+		}
+
+		list($this->strPickerTable, $this->strPickerField, $this->intPickerId) = explode('.', \Input::get('target'), 3);
+		$this->intPickerId = (int) $this->intPickerId;
+
+		\Controller::loadDataContainer($this->strPickerTable);
+
+		if (!isset($GLOBALS['TL_DCA'][$this->strPickerTable]['fields'][$this->strPickerField]))
+		{
+			throw new InternalServerErrorException('Target field "' . $this->strPickerTable . '.' . $this->strPickerField . '" does not exist.');
+		}
```

Comment:
This `if` needs to be moved to the end of the method, because the DCA `onload_callback` may add the field dynamically.